src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -509,6 +511,22 @@ public void testRequireOneOfVCFOrSampleNameFile(){
         runCommandLine(args);
     }
 
+    @Test
+    public void testGenomicsDBImportWithoutDBField() throws IOException {
+	//Test for https://github.com/broadinstitute/gatk/issues/3736
+	final List<String> vcfInputs = Arrays.asList(NA_24385);
+        final String workspace = createTempDir("genomicsdb-tests").getAbsolutePath() + "/workspace";
```

Comment:
I've noticed that the indentation in your PRs is always a little off @psfoley -- for future PRs, you may want to look at your IDE settings. We generally set our IDEs to expand tabs into 4 spaces, which may be the issue here.

(Not a blocker for merge of this PR, however)